qa/suites/upgrade-clients/client-upgrade-quincy-tentacle/tentacle-client-x/rbd/2-workload/rbd_notification_tests.yaml

```diff
@@ -0,0 +1,34 @@
+tasks:
+- parallel:
+  - workunit:
+      branch: squid
+      clients:
+        client.0:
+          - rbd/notify_master.sh
+      env:
+        RBD_FEATURES: "61"
+  - workunit:
+      branch: tentacle
+      clients:
+        client.1:
+          - rbd/notify_slave.sh
+      env:
+        RBD_FEATURES: "61"
+        #RBD_DISABLE_UPDATE_FEATURES: "1"
```

Comment:
Get rid of it completely -- this environment variable is no longer recognized so the commented line carries no information.
```suggestion
```